test/test.js

```diff
@@ -127,8 +128,21 @@ describe('hashmap', function() {
 		});
 	});
 
+	describe('hashmap.delete()', function() {
+		it('should delete an entry by key', function() {
+			hashmap.set('key', 'value1');
+			hashmap.delete('key');
+			expect(hashmap.has('key')).to.be.false;
+		});
+
+		it('should not fail when the key is not found', function() {
+			hashmap.delete('key');
+			expect(hashmap.has('key')).to.be.false;
+		});
+	});
+
 	describe('hashmap.remove()', function() {
```

Comment:
Remember to just this whole method by asserting that it's equal to `delete`. Check comment above on how to make it work